src/pykiso/lib/connectors/cc_rtt_segger.py

```diff
@@ -117,7 +117,7 @@ def __init__(
         self.rx_buffer_size = None
         # initialize rtt logging specific parameters
         self.rtt_configured = False
-        self._is_running = False
+        self._is_log_thread_running = False
```

Comment:
Maybe just _logger_thread_running would be enough

Reply:
as you whish.. 
done